src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/furtherevidence/actionfurtherevidence/ActionFurtherEvidenceAboutToSubmitHandlerTest.java

```diff
@@ -195,6 +195,117 @@ public void givenAValidPostponementRequest_thenSscsDocumentTypeIsPostponementReq
                 .anyMatch(note -> note.getValue().getNoteDetail().equals("Request Detail Test")), is(true));
     }
 
+    @Test
+    @Parameters({
```

Comment:
If this class used Junit5 I might recommend changing to Paramaterized with EnumSource of PartyItemList and make the test specific to set aside. Perhaps that would be overboard, I just have Lucas's words ringing in my ears! Anyway, I think this is fine bc it's clear what's going on and is easily extensible to the future post hearing types.

Reply:
Agreed, otherwise we're rewriting large test classes or having a mish-mash of two implementations.